CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
 # Changelog
 
-## UNRELEASED
+## 4.0.0 (2024-03-12)
 
 ### Adds
 
 * Add translation keys used by the multisite assembly module.
 * Add side by side comparison support in AposSchema component.
 * Add the possibility to make widget modals wider, which can be useful for widgets that contain areas taking significant space. See [documentation](https://v3.docs.apostrophecms.org/reference/modules/widget-type.html#options).
 
+### Changes
+
+* Migrate to Vue 3.
```

Comment:
How about:

"Migrate to Vue 3. This entails changes to some admin UI code, as detailed in our public announcement. There are **no other** backwards incompatible changes in `apostrophe` version 4.0.0. Certain other modules containing custom admin UI have also been updated in a new major version to be compatible, as noted in our announcement and on the migration page of our website."